src/Orleans/Runtime/OutsideRuntimeClient.cs

```diff
@@ -153,7 +153,7 @@ public OutsideRuntimeClient(ClientConfiguration cfg, bool secondary = false)
 
                 if (!secondary)
                 {
-                    UnobservedExceptionsHandlerClass.SetUnobservedExceptionHandler(UnhandledException);
+                    UnobservedExceptionsHandlerClass.TrySetUnobservedExceptionHandler(UnhandledException);
```

Comment:
I think it's benign, and I'd happily remove it. Here's the implementation of that handler:
```C#
private void UnhandledException(ISchedulingContext context, Exception exception)
{
    logger.Error(ErrorCode.Runtime_Error_100007, String.Format("OutsideRuntimeClient caught an UnobservedException."), exception);
    logger.Assert(ErrorCode.Runtime_Error_100008, context == null, "context should be not null only inside OrleansRuntime and not on the client.");
}
```
Should I remove it?